pkg/ottl/contexts/internal/ctxprofile/profile.go

```diff
@@ -27,8 +27,6 @@ func PathGetSetter[K Context](path ottl.Path[K]) (ottl.GetSetter[K], error) {
 		return accessSampleType[K](), nil
 	case "sample":
 		return accessSample[K](), nil
-	case "location_indices":
```

Comment:
Could you please remove all dropped fields from all context's docs (`pkg/contexts/<context>/README.md`)?